eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -12,41 +12,46 @@ param (
   [String]$ServiceDirectory,
   [Parameter(Mandatory = $true)]
   [String]$PackageName,
-  [boolean]$Unreleased=$True,
-  [boolean]$ReplaceLatestEntry = $False,
+  [String]$Unreleased=$True,
```

Comment:
If you want these to be strings now we should probably default them to strings instead of booleans.